hasura/.hasura/context.yaml

```diff
@@ -79,3 +79,6 @@ definition:
     docker-stop:
       bash: for file in compose*.yaml $(find ../.data -type f -name "compose.yaml"); do docker compose -f $file down -v; done
       powershell: docker compose -f compose.yaml down -v; docker compose -f .data/compose.yaml down -v
+    jwt-gen:
+      bash: cd ../scripts/jwt && npm i && node ./jwt.mjs --env $(find ../../ -name '.env.cloud.*' -print -quit) -r customer -u 7
```

Comment:
Couple of nitpicks:
- Do we need to do an `npm i` for every command invocation? Users could run that themselves as part of the initial set up
- The jwt script is context aware so we don't need to cd into the directory to invoke

Reply:
I can go either way. I like having it for the cases where I forgot to do it, and it adds milliseconds to the script if it already exists. Can go with your preference.

Reply:
Can I be a pain and ask that we remove the `npm i` so the ddn run just calls the script. The script has uses -r customer -u 7 by default so we could always omit those too for simplicity